contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -3500,6 +3612,9 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 			{
 				DropStmt   *drop_stmt = (DropStmt *) parsetree;
 
+				if (drop_stmt->removeType == OBJECT_TABLE)
```

Comment:
Do we need to add a new function for OBJECT_TABLE, can we use/enhance bbf_ExecDropStmt?

Reply:
`bbf_ExecDropStmt` function lacks the capability to handle drop operations from PSQL dialects, which may result in inconsistencies in the metadata and it will need a separate effort to fix that.